docs/modules/reference/nav.adoc

```diff
@@ -14,4 +14,14 @@
 ** xref:configuration/tuning-activemq.adoc[]
 ** xref:configuration/ttl-rpc.adoc[]
 ** xref:configuration/aws-sqs.adoc[]
+** xref:daemons/introduction.adoc[]
```

Comment:
I would move the Daemons section up to be just under the Reference section as a main topic area.
```suggestion
* xref:daemons/introduction.adoc[]
```